packages/@magic-sdk/react-native-bare/src/react-native-webview-controller.tsx

```diff
@@ -139,6 +140,18 @@ export class ReactNativeWebViewController extends ViewController {
     }, [show]);
 
     const handleWebViewMessage = useCallback((event: any) => {
+      const data = JSON.parse(event.nativeEvent.data);
```

Comment:
Is `event.nativeEvent.data ` always defined? Because otherwise it would result in a crash, and that `event: any` typing above is not very reassuring 🤔

Reply:
It is not, hence the `if-conditional` check below 🤔